CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 3.13...3.18)
```

Comment:
This will introduce unwanted behavior change regarding cmake policies.
https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html#policy-settings